samples/isilon_v230_k8s_124.yaml

```diff
@@ -182,10 +190,14 @@ spec:
       # Examples:
       #   node-role.kubernetes.io/control-plane: ""
       nodeSelector:
-      #  node-role.kubernetes.io/control-plane: ""
+      # Uncomment if nodes you wish to use have the node-role.kubernetes.io/master taint
+      #  node-role.kubernetes.io/master
```

Comment:
master is not supported in K8s1.24. We may have to remove this for K8s1.24 sample